lua/telescope/pickers.lua

```diff
@@ -339,10 +339,14 @@ function Picker:find()
 
   -- Prompt prefix
   local prompt_prefix = self.prompt_prefix
+  a.nvim_buf_set_option(prompt_bufnr, "buftype", "prompt")
```

Comment:
I would prefer only to set prompt as buftype if we actually have a prefix because prompt buffers are still broken in neovim, because of missing vim patches and you can workaround these issues if you are fine without a prompt prefix.

See https://github.com/nvim-telescope/telescope.nvim/labels/upstream